MediaWikiKit/MediaWikiKit/MWKArticle.h

```diff
@@ -99,6 +99,25 @@ static const NSInteger kMWKArticleSectionNone = -1;
 - (MWKImage*)importImageURL:(NSString*)url sectionId:(int)sectionId;
 
 /**
+ *  Add the given URL to the receiver's image list, as well as to the corresponding section's image list.
+ *
+ *  @param sourceURL The URL string to add.
+ *  @param sectionId The section whose image list the URL should also be added to.
+ *  @param skipIfPresent @c YES if the image should only be added if the list doesn't contain it.
+ */
+- (void)updateImageListsWithSourceURL:(NSString*)sourceURL inSection:(int)sectionId skipIfPresent:(BOOL)skipIfPresent;
```

Comment:
Seems this should be "add" instead of "update" - I suspected thats what it did, but had to read the comments to be sure what this method was doing.